crates/playground/js/wasm.worker.ts

```diff
@@ -0,0 +1,22 @@
+import * as Comlink from "comlink";
+
+const initHandlers = async () => {
+  const rust_wasm = await import("../pkg");
+  await rust_wasm.default();
+  const numThreads = navigator.hardwareConcurrency;
+  // must be included to init rayon thread pool with web workers
+  await rust_wasm.initThreadPool(numThreads);
+  return Comlink.proxy({
+    sim: null,
+    numThreads: numThreads,
```

Comment:
`numThreads: numThreads,` -> `numThreads`
Maybe we can capture something like this with ESlint in CI in the future.